python/llm/example/CPU/LangChain/README.md

```diff
@@ -1,90 +1,148 @@
-## Langchain Examples
+# Langchain Example
 
-This folder contains examples showcasing how to use `langchain` with `ipex-llm`. 
+The examples in this folder shows how to use [LangChain](https://www.langchain.com/) with `ipex-llm` on Intel CPU.
 
-### Install-IPEX LLM
+> [!NOTE]
+> Please refer [here](https://python.langchain.com/docs/integrations/llms/ipex_llm) for upstream langchain documentation with ipex-llm.
```

Comment:
Add documentations reference for both upstream llms and embeddings.

langchain -> LangChain